packages/editor/src/components/post-publish-button/label.js

```diff
@@ -21,13 +22,13 @@ export function PublishButtonLabel( {
 } ) {
 	if ( isPublishing ) {
 		return __( 'Publishing…' );
-	} else if ( isPublished && isSaving && ! isAutosaving ) {
+	} else if ( ( isPublished || isScheduled ) && isSaving && ! isAutosaving ) {
```

Comment:
This doesn't work as expected:

* Edit Post => Click "Save Draft" => Label "Publish" ✅ 
* Edit Post + change date to future + click "Publish" => Label "Scheduling…" ❌
* Click "Schedule" => Label "Updating…" ❌
* Click "Switch to draft" => Label "Scheduling…" ❌
* Reset publish date + click "Publish" => Label "Updating…" ❌